### ISSUE TITLE
Requiring git commit breaks 

### ISSUE THREAD

**Issue by abenson** [nvarner/typst-lsp#304]
- **Component**: 
    - LSP (used with other editor)
- **LSP version**: 0.10.0
- **OS version and name**: Void Linux

- I am on the [latest](https://github.com/nvarner/typst-lsp/tags) stable version of the extension/LSP.
- I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.

## Issue

The latest build includes the `git` commit-hash as part of the version. This breaks building from release tarballs.

### Logs

```
error: failed to run custom build command for `typst-lsp v0.10.0 (/builddir/typst-lsp-0.10.0)`

Caused by:
  process didn't exit successfully: `/builddir/typst-lsp-0.10.0/target/release/build/typst-lsp-b514d2c952a71ab3/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "command 'git rev-parse HEAD' failed: exit=128 stdout='' stderr='fatal: not a git repository (or any of the parent directories): .git\\n'"', build.rs:81:45
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

**Comment by KillTheMule** [comment]
Should be fixed by https://github.com/nvarner/typst-lsp/pull/303

**Comment by abenson** [comment]
Fixed in v0.10.1 / 82cfb64.
